exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -202,7 +214,8 @@ function ser:serialize-empty-params() {
 };
 
 declare
-    %test:assertEquals("aaa bbb")
+    %test:assertEquals("aaa
```

Comment:
are you sure about this?  saxon: 
<img width="719" alt="image" src="https://user-images.githubusercontent.com/1700062/175789006-d64f33e4-fcad-4730-baf3-b0c67c7060d6.png">

Reply:
Indeed, unless I'm misreading it, [the XSLT & XQuery serialization spec](https://www.w3.org/TR/xslt-xquery-serialization-31/#:~:text=each%20separated%20by-,a%20single%20space) states that the default item-separator is a single space:

> If the item-separator serialization parameter is absent, then for each subsequence of adjacent strings in S2, copy a single string to the new sequence equal to the values of the strings in the subsequence concatenated in order, each separated by **a single space**.